packages/cdktf-cli/templates/java/src/main/java/com/mycompany/app/MainStack.java

```diff
@@ -0,0 +1,14 @@
+package com.mycompany.app;
+
+import software.constructs.Construct;
+
+import com.hashicorp.cdktf.TerraformStack;
+
+public class Main extends TerraformStack
```

Comment:
Should we name this "MyStack" to be similar to how our TypeScript template looks?

Reply:
Either `MyStack` or `MainStack` would be better than `Main`. Also need to make sure the class name matches the file name.